### ISSUE TITLE
Create fedora-messaging schema for the post-receive-fedora-messaging distgit pagure hook

### ISSUE THREAD

**Issue by abompard** [fedora-infra/fedora-messaging-schemas-issues#8]

**Comment by ryanlerch** [comment]
Need some more information about the specific application for this one. It may refer to https://pagure.io/fedscm-admin , but that tool does not seem to send any messages.

**Comment by ryanlerch** [comment]
@pypingou any idea what this one could be?

**Comment by pypingou** [comment]
These are basically the messages sent by a git hook upon commit pushed to dist-git.

https://github.com/fedora-infra/fedmsg_meta_fedora_infrastructure/blob/develop/fedmsg_meta_fedora_infrastructure/scm.py

https://pagure.io/fedora-infra/ansible/blob/master/f/roles/git/hooks/files/post-receive-fedora-messaging

**Comment by ryanlerch** [comment]
OK, it looks like this script only pushes 1 message.

So far, we have been creating a new project/python package for the schemas, do we want to have something of that scale for this single message?

Also, does dist-git send any other messages? just a little unsure how to approach this one.

any ideas @abompard ?

**Comment by abompard** [comment]
It depends, who and which apps are interested in this message? I suppose we could have a `fedora-infra-misc-messages` package for the small things, but I'm afraid it'll eventually turn into `fedmsg_meta_fedora_infrastructure`, so we need to be careful... Maybe put this one at the bottom of the queue with other small scripts and we'll be able to evaluate how many of them we have in this situation?

**Comment by pypingou** [comment]
>    Also, does dist-git send any other messages? just a little unsure how to
>    approach this one.

The dist-git "app" sends notification on new uploads to the lookaside cache.

**Comment by pypingou** [comment]
>    OK, it looks like this script only pushes 1 message.
> 
>    So far, we have been creating a new project/python package for the
>    schemas, do we want to have something of that scale for this single
>    message?

Hm the git hook that sends these messages is in ansible and the fedora infra's
ansible repo also sends messages. Should we bundle the schemas in the same
project?

**Comment by abompard** [comment]
We could, it just means that consuming apps (like the future FMN) will have to install everything just to get the schemas. As long as there aren't too many dependencies I guess it's fine (looks like the main one is pygit2).

**Comment by pypingou** [comment]
Hm, from your answer I'm not sure that I was clear.
I meant the same ansible-infra-schemas project or so. What did you have in mind?

**Comment by abompard** [comment]
Oh, right, no I misunderstood. I didn't realize you were talking about two different scripts. Yeah I guess those schemas could both be in an ansible-infra-schemas repo.

**Comment by ryanlerch** [comment]
**INFO:** The ansible callback plugin messages are tracked in this ticket: https://github.com/fedora-infra/fedora-messaging-schemas-issues/issues/28

**Comment by ryanlerch** [comment]
Yeah, it makes sense for an "ansible-infra-schemas" python package / project here.

**Comment by ryanlerch** [comment]
Here is an example of the message this script sends:

https://apps.fedoraproject.org/datagrepper/id?id=2020-f4243f62-e22c-4fe2-8004-d985300bb153&is_raw=true&size=extra-large

**Comment by pypingou** [comment]
Discussing with @adamsaleh we renamed Randy's project fedora-messaging-infragit to fedora-infra-messages https://github.com/fedora-infra/fedora-infra-messages/ so it can hosts schemas for the ansible callback, git hook for the ansible repo and the git hook for dist-git. The sources for all three being in our ansible repo.

**Comment by pypingou** [comment]
Actually there is a PR pending review at: https://github.com/fedora-infra/fedorainfra-ansible-messages/pull/2/
